Makefile

```diff
@@ -133,12 +133,10 @@ docker: _print_vars
 	docker build -t $(IMG) -f ./build/Dockerfile .
 	docker push $(IMG)
 
-### webhook: generate certificates for webhooks and deploy to cluster; no-op if webhooks are disabled or running on OpenShift
+### webhook: generate certificates for webhooks and deploy to cluster; no-op if running on OpenShift
 webhook:
-ifeq ($(WEBHOOK_ENABLED),true)
 ifeq ($(TOOL),kubectl)
 	./deploy/webhook-server-certs/deploy-webhook-server-certs.sh kubectl
-endif
 else
 	@echo "Webhooks disabled, skipping certificate generation"
```

Comment:
This message going to appear when deploying on OpenShift and it seems not to be actual anymore.